docs/benchmarks/micro-benchmarks.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
The display of id is by designed? It's a little weird: [micro-benchmarks.md](https://github.com/microsoft/superbenchmark/blob/33414336d2a727111482c2ed8c99f593c033ffda/docs/benchmarks/micro-benchmarks.md)

Reply:
it's markdown frontmatter, [for example](https://jekyllrb.com/docs/front-matter/)
this is doc source, not meant to be viewed by user directly, like [other microsoft docs](https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/service-fabric/service-fabric-cli.md)

Reply:
The term 'id' here is weird to me. Can we use 'title'?

Reply:
`id` is a field used when generating layout for website
there does exist a `title` field, by default it's the first header's content